src/App/Repository/HXLLicenseRepository.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/**
+ * Ushahidi Config Repository, using Kohana::$config
+ *
+ * @author     Ushahidi Team <team@ushahidi.com>
+ * @package    Ushahidi\Application
+ * @copyright  2014 Ushahidi
+ * @license    https://www.gnu.org/licenses/agpl-3.0.html GNU Affero General Public License Version 3 (AGPL3)
+ */
+
+namespace Ushahidi\App\Repository;
+
+use Ushahidi\Core\Entity\HXLLicense;
+use Ushahidi\Core\Entity\HXLLicenseRepository as HXLLicenseRepositoryContract;
+use Ushahidi\Core\SearchData;
+use Ushahidi\Core\Usecase\ReadRepository;
+use Ushahidi\Core\Usecase\SearchRepository;
+
+class HXLLicenseRepository extends OhanzeeRepository implements
+	HXLLicenseRepositoryContract,
+	ReadRepository,
```

Comment:
HXLLicenseRepositoryContract should probably just extend the Read and Search repo interfaces itself.